src/core/qgis.cpp

```diff
@@ -306,7 +309,11 @@ QString Qgis::geosVersion()
 
 int Qgis::geosVersionInt()
 {
-  return QStringLiteral( "%1%2%3" ).arg( GEOS_VERSION_MAJOR, 2, 10, QChar( '0' ) ).arg( GEOS_VERSION_MINOR, 2, 10, QChar( '0' ) ).arg( GEOS_VERSION_PATCH, 2, 10, QChar( '0' ) ).toInt();
+  return QStringLiteral( "%1%2%3" )
+         .arg( GEOS_VERSION_MAJOR, 2, 10, QChar( '0' ) )
+         .arg( GEOS_VERSION_MINOR, 2, 10, QChar( '0' ) )
+         .arg( geosVersionPatch(), 2, 10, QChar( '0' ) ).toInt()
+         ;
```

Comment:
```suggestion
  static const int version = QStringLiteral( "%1%2%3" )
         .arg( GEOS_VERSION_MAJOR, 2, 10, QChar( '0' ) )
         .arg( GEOS_VERSION_MINOR, 2, 10, QChar( '0' ) )
         .arg( geosVersionPatch(), 2, 10, QChar( '0' ) ).toInt()
         ;
         return version;
```

Reply:
Indentation was made by prepare_commit.sh, and passed CI style checks.
Do you really think that saving the per-process function call is worth the trouble of finding a way to to numbers-from-string extraction in the preprocessor ? I'd be happy with whawt we have here

Reply:
Ah, I see the suggested change, sorry I didn't notice before.
Ok, will do, and also for geosVersionPatch() -- pity, as CI finally passed :)

Reply:
Pushed. @m-kuhn can you see if you like it now ?